examples/resources/bowtie_dns_block_list/resource.tf

```diff
@@ -0,0 +1,9 @@
+# Use a third-party DNS blocklist for content blocking:
+
+resource "bowtie_dns_block_list" "example" {
+  name              = "Infosec Block List"
+  upstream          = "https://raw.githubusercontent.com/hagezi/dns-blocklists/main/domains/tif.txt"
+  override_to_allow = <<EOF
```

Comment:
I would probably make this a list and just render it correctly before sending the request. Maybe even a set to make sure each value shows up once and you don't have order dependence

Reply:
This also means you can modify the payload without requiring everyone to rewrite their payloads with a new format. Assuming it's currently new line delimited

Reply:
Thanks, I think this a good idea. I went ahead and turned this into a list and I think it makes more sense.